aquadoggo/src/graphql/client/static_query.rs

```diff
@@ -27,30 +26,26 @@ impl StaticQuery {
             using the returned arguments"
         )]
         public_key: scalars::PublicKeyScalar,
+        #[graphql(
+            name = "documentViewId",
+            desc = "Document the entry's UPDATE or DELETE operation is referring to, \
+            can be left empty when it is a CREATE operation"
+        )]
+        document_view_id: Option<scalars::DocumentViewIdScalar>,
+        // @TODO: Figure out why this fixes things....
         #[graphql(
             name = "documentId",
             desc = "Document the entry's UPDATE or DELETE operation is referring to, \
             can be left empty when it is a CREATE operation"
         )]
```

Comment:
Taking out `documentId` here breaks something inside the graphql magic, even though we are already using `documentViewId` in all cases.

Reply:
See issue: https://github.com/p2panda/aquadoggo/issues/242